app/grandchallenge/cases/serializers.py

```diff
@@ -193,11 +214,13 @@ def validate(self, attrs):
             )
 
         if "interface" in attrs and not (
-            "archive" in attrs or "archive_item" in attrs
+            "archive" in attrs
+            or "archive_item" in attrs
+            or "display_set" in attrs
         ):
             raise ValidationError(
-                "An interface can only be defined for archive "
-                "or archive item uploads."
+                "An interface can only be defined for archive, "
+                "archive item or display set uploads."
             )
 
         if "archive_item" in attrs and "interface" not in attrs:
```

Comment:
If a `display_set` is defined, an `interface` also needs to be defined, right? Or does it make sense to have default (`generic-medical-image`)? A test for this would also be good.